common/app/implicits/Requests.scala

```diff
@@ -24,7 +24,7 @@ trait Requests {
 
     lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || (!r.host.isEmpty && r.host == Configuration.amp.host)
 
-    lazy val isEmail: Boolean = (r.getQueryString("format") == Some("email")) || r.path.endsWith(EMAIL_SUFFIX)
+    lazy val isEmail: Boolean = r.getQueryString("format").contains("email") || r.path.endsWith(EMAIL_SUFFIX)
```

Comment:
oh yes!  didn't realise that so thanks for the pointer!  Although that doesn't stop my 3rd example above, although we could just hope that noone does that as it would be a little weird!